bin/update_testdata

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env php
+<?php
+
+chdir(__DIR__.'/../');
+shell_exec('rm -rf gherkin_testdata && svn export https://github.com/cucumber/cucumber/trunk/gherkin/testdata gherkin_testdata');
```

Comment:
Rather than copying the gherkin tests from upstream (and not knowing which exact version was copied), I suggest using an inline package repository in the composer.json to make composer install the `cucumber/cucumber` repository as a dev requirement.

See https://github.com/scssphp/scssphp/commit/d6a149e6bb3d54949e234a24d3bc1ed086679873#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34 for a similar approach in scssphp. There, we manage the exact reference we want to use (see https://github.com/scssphp/scssphp/pull/155 for a PR updating the version used)

Reply:
Great technique, thankyou